src/tequila/quantumchemistry/qc_base.py

```diff
@@ -2044,6 +2053,10 @@ def perturbative_f12_correction(self, rdm1: numpy.ndarray = None, rdm2: numpy.nd
                                                    n_ri=n_ri, external_info=external_info, **kwargs)
         return correction.compute()
 
+    def make_transformation(self, operator, U):
```

Comment:
the two lines
```python
H = mol.make_transformation(H,U)
```
and
```python
H = U.dagger()*H*U
```

are the same size. Would propose to just  code it like this, in the two instances you are using this. 
Having this function in the molecule class is probably a bit confusing